tests/test_cli.py

```diff
@@ -181,3 +183,46 @@ def test_bad_config_file_parameter(mock_parse_config, clean_env, parser):
 def test_parse_remote_reset_with_assert(env, parser):
     args = parser.parse_args(["assert", "--remote-reset"])
     assert args.remote_reset
+
+
+def test_logging_failing_explore_sql(tmpdir):
+    error = SqlError(
+        path="example_explore",
+        message="example error message",
+        sql="select example_explore.example_dimension_1 from model",
+    )
+
+    log_failing_sql(error, tmpdir, "explore_model", "example_explore")
+    query_directory = Path(tmpdir / "queries")
+    query_file = Path(query_directory / "explore_model__example_explore.sql")
+    content = open(query_file).read()
+
+    assert os.path.exists(query_directory)
```

Comment:
Let's drop the `os` import and use `Path.exists()` since you're already using pathlib.